keras_hub/src/models/efficientnet/cba.py

```diff
@@ -0,0 +1,139 @@
+import keras
+
+BN_AXIS = 3
+
+CONV_KERNEL_INITIALIZER = {
```

Comment:
Kinda funky to return this as a dict, any particular reason for it?

More in line with other models would be to just add a function here...

```python
def conv_kernel_initializer(scale=2.):
    return keras.initializers.VarianceScaling(
        scale=scale,
        ...
    )
```

Reply:
No particular reason, I was following the original style of the fusedmbconv/mbconv blocks. Done.